### ISSUE TITLE
iOS: Old devices don't reserve space for status bar

### ISSUE THREAD

**Issue by andydotxyz** [fyne-io/fyne#2372]
### Describe the bug:
Status bar on old iOS devices isn't reserved like it should be.

### To Reproduce:
Load any Fyne app on an iOS 10.3 device

### Screenshots:
![IMG_0001](https://user-images.githubusercontent.com/294436/129340441-2127208a-0816-46d9-8121-7c4e5efc66dd.PNG)

### Device (please complete the following information):
 - **OS:** iOS / iPhone 5
 - **Version:** 10.3
 - **Go version:** 1.14
 - **Fyne version:** 2.0.4

**Comment by changkun** [comment]
Are we still targeting iOS 10? It was released six years ago.

**Comment by andydotxyz** [comment]
Yes. It was the last release to support 32bit iOS devices.
I think those should be still alive! They can be built with Go 1.14 which we still support.

**Comment by changkun** [comment]
> Yes. It was the last release to support 32bit iOS devices.
> I think those should be still alive! They can be built with Go 1.14 which we still support.

That's a little bit unfortunate. Working to the support seems to have much less return, as they are deprecating soon.

**Comment by andydotxyz** [comment]
But dropping support means leaving out a host of existing, usable, devices.
Apple has abandoned them (and Go is following them) but I don't think that means we should?...

**Comment by changkun** [comment]
> But dropping support means leaving out a host of existing, usable, devices.
> Apple has abandoned them (and Go is following them) but I don't think that means we should?...

That's true. Dropping support is a little bit sad for people who are still using those devices. However, we don't know how the size of the user group. If they choose to use an unsupported OS (from the manufacturer's perspective), having less software support is probably a price to pay (IMO). Furthermore, from Fyne's perspective, supporting less deprecated devices makes the development process a little bit easier.

(Perhaps in the far future, people may use them for retrospective purposes, but leaving the emulating problem to them is somewhat currently practiced a lot, such as Nintendo)

**Comment by andydotxyz** [comment]
Looking at this now that we will no longer support iOS 32bit I wondered if this is an issue... But iOS 11 still has this problem as well. Unless we want to shift to 12 (2018) as minimum supported it remains an issue...
